crates/languages/src/json/runnables.scm

```diff
@@ -0,0 +1,20 @@
+; Add support `package.json` and `composer.json` script runnable
+
+(
+    (document
+        (object
+            (pair
+                key: (string
+                    (string_content) @_name
+                    (#eq? @_name "scripts")
+                )
+                value: (object
+                    (pair
+                        key: (string (string_content) @run @script)
+                    )
+                )
+            )
+        )
+    ) @package-script
```

Comment:
Not sure about the name, would love to name them `package-script` and `composer-script`. But not sure what you think about having double capture names for the same capture(only for a different name)

Reply:
I think this is the best way forward, yeah; However, it doesn't looks like that capture is being used anywhere. We can probably make it anonymous;
```suggestion
    ) @_script
```
Did you mean that it would be cool to have two separate tags for this? If so, that'd be lovely. You can just use `#set! tag` twice with different arguments and it'll just work.